docs/resources/notebook-authoring-guide.md

```diff
@@ -0,0 +1,147 @@
+# Notebook Authoring Accessibility Guide
+
+Writing notebooks is an art, depending on what as an author you want to share to the world the way that the writing and publishing techniques can vary widely. There are some writing and publishing techniques that you can use to ensure your notebooks' content is more accessible.
+
+In this section, we will introduce some guidelines to make edits to your notebooks for accessibility. It is an excellent step you can take to make our communities more inclusive!
+
+All the information presented is based on the [draft authoring tips](https://github.com/Iota-School/notebooks-for-all/blob/main/resources/event-hackathon/accessibility-tips-for-jupyter-notebooks.md) and the [Notebook authoring accessibility checklist](https://github.com/Iota-School/notebooks-for-all/blob/main/resources/event-hackathon/notebook-authoring-checklist.md) from the Notebooks For All team.
+
+# Table of contents
+1. [Structure](#structure)
+2. [Text](#text)
+3. [Code](#code)
+4. [Media](#media)
+    1. [Images](#images)
+    2. [Visualizations](#visualizations)
+    3. [Videos](#videos)
+    4. [Audio and Sonifications](#audio)
+    5. [Interactive Widgets](#interactive)
+5. [Resources](#resources)
+6. [Acknowledgements](#acknowledgements)
+
+## Structure <a name="structure"></a>
+
+Sighted users are able to perceive structure and relationship using different visual cues. We want to ensure that **information and relationships** are perceivable to all. In order to accomplish this goal, we as content authors need to provide access to the information in different modalities.
+
+We will discuss in this section some recommendations for the most common elements in the structure of your notebooks.
+
+### The First Cell
+
+- [ ] The title of the notebook in a first-level heading (eg. `<h1>` or `# in markdown`).
+- [ ] A brief description of the notebook.
+- [ ] A brief summary of the notebook.
+- [ ] The date first published.
+- [ ] The author(s) and affiliation(s) (if relevant).
+- [ ] The date last edited (if relevant).
+- [ ] A link to the notebook's source(s) (if relevant).
+- [ ] A table of contents in an [ordered list](https://www.markdownguide.org/basic-syntax/#ordered-lists) (`1., 2.,` etc. in Markdown).
+
+### The Rest of the Cells
+
+- [ ] There is only one H1 (`#` in Markdown) used in the notebook.
+- [ ] The notebook uses other [heading tags](https://www.markdownguide.org/basic-syntax/#headings) in order without skipping numbers.
```

Comment:
Little edits for specificity. This has been awkward to write in every iteration, so I'm always experimenting. Open to suggestion.

```suggestion
### Other cells

- [ ] There should be only one H1 (`#` in Markdown) used in the notebook; it should not appear in other cells.
- [ ] The notebook uses other [heading tags](https://www.markdownguide.org/basic-syntax/#headings) in order throughout. This creates a table of contents that supports the whole notebook.
```